install_script_op_worker.sh

```diff
@@ -0,0 +1,553 @@
+#!/bin/bash
+# (C) Datadog, Inc. 2010-present
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ 

```suggestion
# (C) Datadog, Inc. 2023-present
```

Reply:
This was one of those "will probably be part of the template" so I opted not to change it.

Reply:
Yeah, it's not a big deal. Just a bit confusing since it makes it seem like the script existed since 2010. Perhaps it'll end up being one of the templated variables.